### PR TITLE
Plumbing updates

### PR DESCRIPTION
**Pull request by ingeniumed**
**Description**

This PR is focused on plumbing updates mainly. It does the following:

- Set CODEOWNERS to the Bistro squad
- Update the GitHub workflows to use some newer composer actions, taken from the Governance Plugin repo
- Setup bug report and PR templates
- Update the Readme with a TOC, and installation instructions
- Simplify the `composer.json` and update the VIPCS package to 3.0. That includes some cs fixes as a result of it to the code, and to the configuration as well. It's taken from the Governance Plugin which took it from the mu-plugins repo.
- Update the location to the plugin URI, the min wordpress version (this is the minimum to be set for VIP plugins), as well as the version of the plugin itself to 0.3.0.

This is going to prep the repo for the WP-GraphQL upgrade PR. In that, I'll cut the 1.0 release of the plugin as well.

I've tested the plugin and there's no changes to the core workings of it.

### CODE REVIEW

**Review by smithjw1 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,6 +9,63 @@ This plugin bundle provides a number of plugins to help you quickly setup a deco
...
+  - [WPGraphQL Preview](#wpgraphql-preview)
+- [Contributing](#contributing)
+
+## Installation
+
+### Install via `git subtree`
+
+The latest version of the WordPress VIP Decoupled Plugin Bundle is available in the default `trunk` branch of this repository.
+
+We recommend installing the latest plugin version [via `git subtree`][wpvip-plugin-subtrees] within your site's repository:
```
Comment (chriszarate) [inline README.md]: Curious about these recommendations. Also these links don't work

File: settings/settings.php [hunk 2]
```diff
@@ -84,7 +86,7 @@ function is_plugin_enabled( $plugin ) {
  */
 function render_graphql_endpoint() {
 	// Match logic in \WPGraphQL\Router:
-	// https://github.com/wp-graphql/wp-graphql/blob/df2c5b2556fcbd69983d96d4e906572d835c0832/src/Router.php
+	// https://github.com/wp-graphql/wp-graphql/blob/df2c5b2556fcbd69983d96d4e906572d835c0832/src/Router.php.
```
Comment (chriszarate) [inline settings/settings.php:89]: this can be a little disruptive, what is forcing this?

Comment (ingeniumed) [reply settings/settings.php:89]: This is coming from the default OOB coding standards. We could turn this off, but having had this in the Block Data API and Block Governance plugins it's become a force of habit now. It is disruptive at first for sure.

File: README.md (hunk 1 above)
Comment (ingeniumed) [reply README.md]: Just updated it, and simplified it down

File: settings/settings.php (hunk 2 above)
Comment (chriszarate) [reply settings/settings.php:89]: I only mean specifically for this link. When I click on it, it 404s because of the trailing .

**Review by chriszarate — APPROVED** [review]

File: settings/settings.php (hunk 2 above)
Comment (ingeniumed) [reply settings/settings.php:89]: Ah, I see. I'll leave this rule as is as it keeps it consistent with the other plugin, but if it gets disruptive we can revert it.

**Pull request merged**